sycl/test-e2e/ProgramManager/uneven_kernel_split.cpp

```diff
@@ -0,0 +1,45 @@
+// REQUIRES: gpu, TEMPORARY_DISABLED
```

Comment:
This is disabled due to a test issue, agreed to have this fixed and enabled in a follow-up PR.